ansible/idr-ms-image-region.yml

```diff
@@ -0,0 +1,23 @@
+- hosts: "{{ idr_environment | default('idr') }}-omeroreadonly-hosts"
```

Comment:
Any reason not to have this logic inside `idr-omero-readonly.yml` since the target are the OMERO read-only hosts only?

Otherwise, this playbook will need to be added to the top-level numbered playbook so that it gets included in future deployment